src/pointer-analysis/value_set_dereference.h

```diff
@@ -92,10 +92,6 @@ class value_set_dereferencet
     const typet &object_type,
     const typet &dereference_type) const;
 
-  void offset_sum(
-    exprt &dest,
-    const exprt &offset) const;
-
```

Comment:
Ok, but this doesn't quite fit what the title of the PR says?!

Reply:
If the PR title says "symex" and I find this change without much of a comment then I have to call it out as a reviewer: based on the information provided in the title, I must assume that it was accidentally included.

Reply:
Oh right I see, sorry, I will be more careful next time